resources/js/Pages/City/Index.vue

```diff
@@ -107,7 +131,17 @@ function createOpinion() {
             {{ city.latitude }}, {{ city.longitude }}
           </h2>
           <ProviderIcons class="pt-4" :item="city" :providers="props.providers" />
-
+          <div class="regulations relative overflow-hidden rounded border-[1px] border-solid border-gray-200 px-3">
```

Comment:
```suggestion
          <div class="regulations relative overflow-hidden rounded border border-solid border-gray-200 px-3">
```

`border` class means `border-width: 1px;`, so we don't have to write `border-[1px]`